modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/action/management/v1/ActionsFailureTest.java

```diff
@@ -0,0 +1,227 @@
+package org.wso2.identity.integration.test.rest.api.server.action.management.v1;
```

Comment:
Add licenses

Reply:
Fixed with: https://github.com/wso2/product-is/pull/20663